command/job_status.go

```diff
@@ -169,7 +169,7 @@ func (c *JobStatusCommand) Run(args []string) int {
 	basic := []string{
 		fmt.Sprintf("ID|%s", *job.ID),
 		fmt.Sprintf("Name|%s", *job.Name),
-		fmt.Sprintf("Submit Date|%s", formatTime(getSubmitTime(job))),
+		fmt.Sprintf("Submit Date|%s", formatTime(time.Unix(0, *job.SubmitTime))),
```

Comment:
I like the utility function here, as it's not obvious that `job.Submit` is in nano seconds.